projects/novo-elements/src/elements/chips/ChipInput.ts

```diff
@@ -117,9 +127,11 @@ export class NovoChipInput implements NovoChipTextControl, OnChanges {
   }
 
   /** Checks to see if the blur should emit the (chipEnd) event. */
-  _blur() {
+  _blur(blurEvt: FocusEvent) {
```

Comment:
its ok to keep it as-is. But i feel like naming the param `blurEvent` would have been a little more clear for only 2 extra characters